cirq/ion/ion_decomposition.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
You are mixing later changes into this PR, which should specifically be about XXPowGate. Open a separate PR (or followup PR) for these changes.